core/src/main/java/org/apache/spark/memory/TaskMemoryManager.java

```diff
@@ -133,7 +133,7 @@ public TaskMemoryManager(MemoryManager memoryManager, long taskAttemptId) {
    * Acquire N bytes of memory for a consumer. If there is no enough memory, it will call
    * spill() of consumers to release more memory.
    *
-   * @return number of bytes successfully granted (<= N).
+   * @return number of bytes successfully granted (>= N).
```

Comment:
https://github.com/apache/spark/blob/af1dba7ca501fd9372b158793119163e3fcd1f24/core/src/main/java/org/apache/spark/memory/TaskMemoryManager.java#L219-L220
^^^ It's not true if we stop the spill when there's nothing to spill, right?

Reply:
You are right. Let's just remove the comparison with N here...